client/js/components/procedure/basicSettings/AutoSwitchProcedurePhaseForm.vue

```diff
@@ -64,10 +64,7 @@
             enforce-plausible-dates
             :min-date="startDate"
             required
-            :data-cy="{
-                endDate: dataCyEndDate,
-                startDate: dataCyStartDate
```

Comment:
As we're using `AutoSwitchPrcedurePhaseForm` in two places, we must preserve the possibility to pass a unique value here. The differentiation between start- and enddate is handled by DpDateRangePicker and can be removed here, so i'd suggest to just have one prop `dataCy` which could then receive the values of `administrationEditForm:procedurePhasePeriod` and `administrationEditForm:procedurePhasePeriodPublic` (or something similar) from where the component is used.

Reply:
so you mean smth. like that: https://github.com/demos-europe/demosplan-core/pull/3089/commits/122220aad2f1df711b48358ec63645467443a77a ?

the result in DpDatepicker will be `administrationEditForm:procedurePhasePeriod:startDate` and `administrationEditForm:procedurePhasePeriod:endDate`. Is it okay to have such a long name, or would it be better to use smth. like `procedurePhasePeriod:startDate` and `procedurePhasePeriod:endDate `

Reply:
after merging the main branch into mine I realized that it was done already, so my comment is outdated....